pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -398,6 +398,18 @@ protected void internalCreatePartitionedTopic(int numPartitions, boolean authori
         }
     }
 
+    protected void internalCreateNonPartitionedTopic(boolean authoritative) {
+    	validateAdminAccessForTenant(topicName.getTenant());
+    	
+    	try {
+    		PersistentTopic topic = (PersistentTopic) getOrCreateTopic(topicName);
+    		//...
```

Comment:
what is this for?

Reply:
Sorry about that, that was meant to denote that there should be some lines of code below it. (which I will probably add later)